raw-ipa-lib/src/helpers/aggregation.rs

```diff
@@ -59,6 +80,16 @@ impl Helper {
         fs::write(f, serde_json::to_string(&self)?.as_bytes())?;
         Ok(())
     }
+
+    pub fn sum<'item, const N: u32>(
+        self,
+        shares: impl IntoIterator<Item = (AdditiveShare<N>, &'item EncryptedSecret)>,
+    ) -> AdditiveShare<N> {
+        shares
+            .into_iter()
+            .map(|(share, secret)| self.share_decryption.decryptor(secret).decrypt(share))
```

Comment:
I'm trying, but I just can't get used to this use of the term "secret". I associate this word with "secret key" and think of it as a value that cannot be shared. To call a value which is distributed to others a "secret" just just really confusing to me. Maybe I'm alone in this, but I'd prefer to name it something else. I should probably have left this feedback on the last pull request. I think that in Diffie-Helman the "shared-secret" is something both parties can ultimately derive that nobody else can see - and doesn't refer to the value derived from the public key that's being passed around in the clear. I'm not sure what to call that part though.

Reply:
Yeah, I was not happy with this either.